service/controller/clusterapi/machine_deployment.go

```diff
@@ -24,9 +24,12 @@ type MachineDeploymentConfig struct {
 	K8sExtClient apiextensionsclient.Interface
 	Logger       micrologger.Logger
 
-	HostAWSConfig  aws.Config
-	ProjectName    string
-	Route53Enabled bool
+	EncrypterBackend string
+	HostAWSConfig    aws.Config
+	InstallationName string
+	ProjectName      string
+	Route53Enabled   bool
+	VaultAddress     string
```

Comment:
There are more information necessary to be configured for the encryption resource.